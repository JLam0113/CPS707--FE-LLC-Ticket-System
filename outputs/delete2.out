Welcome, please enter a command.
Please enter your username
Login unsuccessful (user not found). Please enter your username.
Privileged Login successful, now logged in as admin, please enter a command
Session ended
