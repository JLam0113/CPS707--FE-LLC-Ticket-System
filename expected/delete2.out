Welcome, please enter a command.
Please enter your username
Privileged Login successful, now logged in as admin, please enter a command
Delete account selected, please enter the username:
Transaction cancelled (user is invalid), please enter a command
Session ended
